blog-post-review.md

```diff
@@ -0,0 +1,261 @@
+- [Jupyter](https://jupyter.org/install) - using jupyter-notebook you can write  `CSV` importing and graph analytics in one file
+- [GQLAlchemy](https://pypi.org/project/gqlalchemy/) - to connect to Memgraph with Python
+- [Memgraph Lab](https://memgraph.com/lab) - a GUI tool we use to to visualize graphs
+
+
+### Memgraph with Docker
+We use *[Docker](https://www.docker.com/)* because the Memgraph database is a native Linux application and cannot be installed on Windows and macOS. But for this part, we will use **Docker image** of  `mage-cugraph`. [Here](https://memgraph.com/docs/mage) you can check how to pull the image, and set everything up before running the **Memgraph-cuGraph** container.
+
+
+Before running `mage-cugraph` Docker image, you should [clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) **[jupyter-memgraph-tutorials](https://github.com/memgraph/jupyter-memgraph-tutorials)** GitHub repo and position yourself inside `jupyter-memgraph-tutorials/cugraph-analytics` folder. We will do one "hack" here for later. Since we are working with *Docker* and we will read our **CSV** files, **Memgraph** needs to have access to such files. But they are currently on our machine and need to be inside *Docker*. That's we will create **Docker** volume mounting our current `data/facebook_clean_data/` folder to `/samples` folder inside the Docker container. Inside `/samples` folder, *Memgraph* will find our **CSV** files.
+
+After you cloned the repo and positioned yourself in it, now we can start the `mage-cugraph` image:
+
+```
+docker run -it -p 7687:7687 -p 7444:7444 --volume ./data/facebook_clean_data/:/samples mage-cugraph
+```
+
+
+### Jupyter notebook
+After you have Memgraph running, you should install **Jupyter**. For this tutorial, we used **jupyter-lab** and you can install it with the following command:
+```
+pip install jupyterlab
+```
+
+Now you can go and download **[jupyter-notebook](https://github.com/memgraph/jupyter-memgraph-tutorials/tree/main/cugraph-analytics)** we use in our tutorial. Note, in a repository folder `cugraph-analytics/data` there are `CSV` files containing the full dataset which we will use later in the tutorial.
+
+
+### GQLAlchemy installation
+We will be using the **[GQLAlchemy](https://memgraph.com/docs/gqlalchemy/)** object graph mapper (OGM) to connect to Memgraph and execute **Cypher** queries from **Python**. You can think of Cypher as SQL for graph databases. It contains many of the same language constructs like `CREATE`, `UPDATE`, `DELETE`... and it's used to query the database. GQLAlchemy serves as a Python driver/client for Memgraph. Go to our [installation](https://memgraph.com/docs/gqlalchemy/installation) page to check how more options on how to install it. If you wish, you can install it with `pip`, but be sure also to have `CMake` installed. After you have `CMake` ready, you can install `GQLAlchemy` with the following command:
```

Comment:
GQLAlchemy is an object graph mapper (OGM) used to connect to Memgrapha and execute queries using Python.
...the same language constructs such as  `CREATE`, `UPDATE`, `DELETE`, etc.

I think you should lose the "and it's used to query the database" because I think it implies that if you execute queries, you are querying the database, so you can continue that sentence with:
The sentence "GQLAlchemy serves as a Python driver/client for Memgraph." also seems like repeating what is already said in the first sentence.

Go to the GQLAlachemy installation page for installation instructions and more information. If you have CMake, the easiest way to install GQLAlchemy is with `pip`:

COMAND